app/config/services.yml

```
@@ -191,6 +191,7 @@ services:
     Graby\Graby:
         arguments:
             $config:
+                http_client: { "ua_browser": "Wallabag (Guzzle/5)" }
```

```suggestion
                http_client:
                    ua_browser: "%wallabag.user_agent%"
```
better use a parameter IMO